.travis.yml

```diff
@@ -18,8 +18,8 @@ jobs:
       deploy:
           on:
             branch: master
-            tags: true
+            condition: $TRAVIS_COMMIT_MESSAGE =~ ^chore\(release\):\ ([0-9]{1,3}).([0-9]{1,3}).([0-9]{1,3})$
```

Comment:
what if there if a prerelease ? e.g. "1.0.4-01" ? this looks like it will fail

Reply:
that's a good catch, thank you

Reply:
https://regexr.com/4u5a1 updated this cf43ba3